storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -14,7 +14,7 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 (ns backtype.storm.daemon.supervisor
-  (:import [java.io OutputStreamWriter BufferedWriter IOException])
+  (:import [java.io OutputStreamWriter BufferedWriter IOException FileOutputStream])
```

Comment:
* OutputStreamWriter and BufferedWriter appear unused, we could remove them now.
* Below, we import File from java.io, we can consolidate the remaining two on this line.